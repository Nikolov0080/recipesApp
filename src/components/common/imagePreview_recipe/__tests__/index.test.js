import React from 'react'
import ReactDOM from 'react-dom';
import Input from '../index';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('renders OK', () => {
    const div = document.createElement("div");
    ReactDOM.render(<Input/>,div);
    ReactDOM.unmountComponentAtNode(div);
});

it('matches snapshot inputImage2',()=>{
    const three = renderer.create(<Input  />).toJSON();
    expect(three).toMatchSnapshot();
})