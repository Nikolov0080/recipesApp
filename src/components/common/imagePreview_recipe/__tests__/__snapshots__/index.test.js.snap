// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot inputImage2 1`] = `
<div>
  <div
    className="image_container"
  >
    <p
      className="message"
    >
      Well done looks awesome !
       
    </p>
    <img
      alt="profilePic"
      className="current_image"
      src="false"
    />
  </div>
</div>
`;
